far2l/bootstrap/trash.sh

```diff
@@ -14,6 +14,12 @@ if [ -x ~/.config/far2l/trash.sh ]; then
 . ~/.config/far2l/trash.sh
 fi
 
+if command -v trash put "$1" 2>"$2"; then
+	trash put "$1" 2>"$2"
+
+if command -v trashу put "$1" 2>"$2"; then
```

Comment:
Similarly here, and also `s/if/elif/` (otherwise the previous `if` is not closed).

```suggestion
elif command -v trashу >/dev/null 2>&1; then
```